src/components/Table/table-cells/TableCell.scss

```diff
@@ -0,0 +1,26 @@
+.moonstone-TableCell {
+    padding-right: var(--spacing-medium);
+
+    &:first-child {
+        padding-left: var(--spacing-medium);
+    }
+
+    .moonstone-TableCell_children {
+        display: -webkit-box;
+        -webkit-line-clamp: 1;
+        -webkit-box-orient: vertical;
```

Comment:
As we have autoprefixer now, we don't need to prefix our CSS properties. What about non-webkit browser?

Reply:
`line-clamp` requires it 🤷🏾‍♀️